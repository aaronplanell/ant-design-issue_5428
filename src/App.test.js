import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';

jest.mock('rc-tabs/lib/ScrollableInkTabBar.js', () => {
  return (props) => <div>{props.children}</div>;
});

jest.mock('react-dom', () => ({
  findDOMNode() { return null; },
  render() { return null; }
}));

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('snapshot matchs', () => {
  const tree = renderer.create(
    <App />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
