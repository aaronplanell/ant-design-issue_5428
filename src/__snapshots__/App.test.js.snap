exports[`test snapshot matchs 1`] = `
<div
  className="App">
  <div
    className="ant-tabs ant-tabs-top ant-tabs-line"
    style={Object {}}>
    <div />
    <div
      className="ant-tabs-content ant-tabs-content-animated"
      style={
        Object {
          "marginLeft": "0%",
        }
      }>
      <div
        aria-hidden="false"
        className="ant-tabs-tabpane ant-tabs-tabpane-active"
        role="tabpanel"
        style={undefined}>
        Content of Tab Pane 1
      </div>
      <div
        aria-hidden="true"
        className="ant-tabs-tabpane ant-tabs-tabpane-inactive"
        role="tabpanel"
        style={undefined} />
      <div
        aria-hidden="true"
        className="ant-tabs-tabpane ant-tabs-tabpane-inactive"
        role="tabpanel"
        style={undefined} />
    </div>
  </div>
</div>
`;
